polylogue/storage/sqlite/migrations/source/021_widen_origin_check_claude_design.sql

```
@@ -0,0 +1,291 @@
+    validation_status, validation_error, validation_drift_count, validation_mode,
+    detection_warnings_json, logical_source_key, revision_kind, source_revision,
+    predecessor_source_revision, predecessor_raw_id, baseline_raw_id,
+    append_start_offset, append_end_offset, acquisition_generation,
+    revision_authority, revision_authority_evidence
+FROM raw_sessions;
+
+DROP TABLE raw_sessions;
+ALTER TABLE raw_sessions__021ff RENAME TO raw_sessions;
+
+CREATE INDEX idx_raw_sessions_origin ON raw_sessions(origin);
+CREATE INDEX idx_raw_sessions_origin_native ON raw_sessions(origin, native_id) WHERE native_id IS NOT NULL;
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)